Project

emily0906 opened the pull request:
I need to make the project for my school

tmurakam committed:
update gradle wrapper 4.6

tmurakam committed:
update: target sdk 28, gradle tools 3.2.1.

tmurakam committed:
update dependencies: firebase, etc.

tmurakam committed:
next development cycle: 2.0.2

tmurakam committed:
fix: add admob app ID in AndroidManifest.xml

tmurakam committed:
minor fix

tmurakam committed:
fix: support library version 統一

tmurakam committed:
update gradle wrapper, gradle plugin.

tmurakam committed:
update: gradle tools 3.4.0, gradle 4.x -> 5.2.1, fabric 1.28.0

tmurakam committed:
update: gradle 5.2.1 -> 5.6.4

tmurakam committed:
update: gradle tools 3.4.0 -> 3.6.0, etc.

tmurakam committed:
refactor: migrate to androidx

tmurakam committed:
update: update firebase, crashlytics, update minSdkVersion 15 -> 16

tmurakam committed:
update: dependencies

tmurakam committed:
update: androidx brower, set java 8

tmurakam committed:
update: Android Studio 4.0

- compileSdkVersion 28 -> 29
- gradle tools 3.6.0 -> 4.0.0
- build tools 28.0.3 -> 29.0.3
- gradle 5.6.4 -> 6.1.1

tmurakam committed:
update firebase

tmurakam committed:
update: migrate to firebase crashlytics from fabric.

tmurakam committed:
update ChangeLog

tmurakam committed:
improve: fix layouts

tmurakam commented:
I don't merge to master branch at this moment, because the new version is not released yet.
The master branch only contains released version.

tmurakam committed:
update: Gradle tools, google-services, firebase.

tmurakam committed:
update dependencies

tmurakam committed:
update: gradle 6.5

tmurakam committed:
update: Gradle 6.5 -> 6.7.1

tmurakam committed:
update: gradle tools 4.1.3 -> 4.2.0

tmurakam committed:
update: sdk 29 -> 30, dependencies

tmurakam committed:
chore: add mavenCentral to remove jcenter

tmurakam committed:
feat: Import TimePickerWithSeconds from https://github.com/IvanKovac/TimePickerWithSeconds

tmurakam committed:
chore: remove old code

tmurakam committed:
bug fix: Avoid overlapping navigation bar

tmurakam committed:
docs: Update changelog

tmurakam committed:
update: Update gradle tools 7.0, java 11, etc.

tmurakam committed:
update: dependencies

tmurakam committed:
update: Update dependencies, minSdkVersion 16 -> 19 (4.4)

tmurakam committed:
fix: add android:export to AndroidManifest.xml

tmurakam committed:
fix: Enable multiDex

tmurakam committed:
update: Upgrade gradle 7.2, plugin 7.1.3

tmurakam committed:
update: firebase, etc

tmurakam committed:
feat: Add kotlin configurations

tmurakam committed:
Rename .java to .kt

tmurakam committed:
feat: Convert BellRinger to kotlin

tmurakam committed:
feat: Convert Prefs to kotlin

tmurakam committed:
Rename .java to .kt

tmurakam committed:
Rename .java to .kt

tmurakam committed:
feat: Convert TimerLogic to kotlin

tmurakam committed:
Rename .java to .kt

tmurakam committed:
feat: Convert FrontFitTextView to kotlin

tmurakam committed:
fix: Remove layout_marginBottom

tmurakam committed:
chore: Use companion object to declare constants

tmurakam committed:
feat: Convert InfoActivity to Kotlin

tmurakam committed:
Rename .java to .kt

tmurakam committed:
Rename .java to .kt

tmurakam committed:
feat: Convert TimeSetActivity to Kotlin

tmurakam committed:
Rename .java to .kt

tmurakam committed:
feat: Convert PrefActivity to Kotlin

tmurakam committed:
feat: Convert MainActivity to Kotlin

tmurakam committed:
Rename .java to .kt

tmurakam committed:
refactor: Use lateinit

tmurakam committed:
refactor: Fix styles

tmurakam committed:
refactor: Fix styles, etc

tmurakam committed:
refactor: Prefs: remove deprecated api call

tmurakam committed:
refactor: Remove deprecated api call for vibration

tmurakam committed:
fix: lateinit Prefs

tmurakam committed:
chore: Update gradle config

tmurakam committed:
fix: Fix crash when click time label

Use setOnClickListener explicitly.

tmurakam committed:
chore: Fix copyright years

tmurakam committed:
refactor: fix inspection results

tmurakam committed:
update: gradle plugin 7.1.3 -> 7.4.2

tmurakam committed:
update: update sdk, dependencies, kotlin 1.8

tmurakam committed:
update: move package (namespace) from manifest to build.gradle

tmurakam committed:
fix: compilation error of adSize

tmurakam committed:
update: Update AGP 8.0, Enable R8 (by AGP Upgrade Assistant)

- Upgrade AGP dependency from 7.4.2 to 8.0.1
- Upgrade Gradle version to 8.0
- Update default R8 processing mode
- Enable buildConfig build feature
- Preserve transitive R classes
- Preserve constant R class values

tmurakam committed:
feat: Rewrite build files based on newly created android project

tmurakam committed:
feat: update AGP 8.0.2

tmurakam committed:
docs: Update ChangeLog

tmurakam committed:
fix: Fix time set activity layout

The checkbox seems don't change state if placed under TimePicker...

tmurakam committed:
chore: fix copyright year

tmurakam committed:
docs: Update ChangeLog 2.1

tmurakam committed:
feat: update v2.1

tmurakam committed:
remove: Remove analytics and ads
